src/components/MentionSuggestions.js

```diff
@@ -63,25 +63,42 @@ const MentionSuggestions = (props) => {
      * @returns {JSX.Element}
      */
     const renderSuggestionMenuItem = (item) => {
-        const displayedText = _.uniq([item.text, item.alternateText]).join(' - ');
-        const styledTextArray = getStyledTextArray(displayedText, props.prefix);
+        const isIcon = item.text === CONST.AUTO_COMPLETE_SUGGESTER.HERE_TEXT;
+        const styledDisplayName = getStyledTextArray(item.text, props.prefix);
+        const styledHandle = getStyledTextArray(item.alternateText, props.prefix);
```

Comment:
https://github.com/Expensify/App/pull/21779/files - bug two same emails display for user without display name